### PR TITLE
playing around with markdown

### PR DESCRIPTION
**Pull request by ebergerrh**

### CODE REVIEW

File: github-eb-demo.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+## Markdown Header
+
+This is a sample github file. 
+
+Time to play with Markdown.
+
+[ ] check
```
Comment (githubteacher) [inline github-eb-demo.md]: @ebergerrh Probably meant to add a `-` before the brackets?

**Comment by ondine** [comment]
:ship: haven't heard of the :shipit: squirrel?!

**Comment by ebergerrh** [comment]
@ondine, me neither. let's use it! :shipit:

**Comment by ebergerrh** [comment]
:rocket:

**Pull request merged**